scalecodec/type_registry/polymesh.json

```diff
@@ -1446,6 +1446,8 @@
   "RefCount": "RefCount",
   "secondary_key::api::LegacyPermissions": "LegacyPermissions",
   "secondary_key::api::secondarykey<accountid>": "SecondaryKey",
-  "<T as IdentityTrait>::OffChainSignature": "OffChainSignature"
+  "<T as IdentityTrait>::OffChainSignature": "OffChainSignature",
+  "ethereum::EcdsaSignature": "EcdsaSignature",
+  "ethereum::EthereumAddress": "EthereumAddress"
```

Comment:
Sorry, I am out of context, but what is `EthereumAddress`?

Reply:
If you go to http://alcyone-explorer-1199142562.ca-central-1.elb.amazonaws.com/POLYMESH/runtime-type?page=3 you will see these are the two type mappings that aren't working.